### ISSUE TITLE
Product Loop / Jet Smart Filters

### ISSUE THREAD

**Issue by Stephen-debug** [dudaster/ele-custom-skin#228]
Hello

I am using your plugin alongside Jet Smart Filters.

I created a Product Loop so I could take greater control of my design, before that I was using the Jet Woo Builder - Products Grid, at that point the Jet Smart Filters were working.

When I moved over to the Product Loop the Jet Smart Filter stopped filtering, I've seen a lot of others on here with a similar problem but none of those threads helped when I followed those instructions.

I'll attach some screenshots (default template name 'Africa' might mislead you but its just the loop name which needs changing.

Hopefully you can help, thanks!

<img width="300" alt="Screenshot 2020-06-05 at 11 50 13" src="https://user-images.githubusercontent.com/66389411/83871266-af031780-a727-11ea-8ce9-09c75f40b015.png">
<img width="294" alt="Screenshot 2020-06-05 at 11 49 10" src="https://user-images.githubusercontent.com/66389411/83871287-b4f8f880-a727-11ea-8518-647c61a830d4.png">
<img width="2025" alt="Screenshot 2020-06-05 at 11 47 40" src="https://user-images.githubusercontent.com/66389411/83871295-b9251600-a727-11ea-8805-b9eda4cc3552.png">
<img width="296" alt="Screenshot 2020-06-05 at 11 49 37" src="https://user-images.githubusercontent.com/66389411/83871305-bcb89d00-a727-11ea-8cd6-22e91353c74e.png">

**Comment by dudaster** [comment]
Does it work with classic skin?

**Comment by Stephen-debug** [comment]
> Does it work with classic skin?

**Comment by Stephen-debug** [comment]
Nope, it doesn't work with the classic skin either.

**Comment by dudaster** [comment]
This mean it isn't an ECS issue. Can you provide an URL?

**Comment by Stephen-debug** [comment]
The site is currently only saved locally using Desktop Server, can you advise the best way for me to share it with you?

**Comment by dudaster** [comment]
Deactivate all the plugins except smart filters and elementor pro and elementor and see if it works.

**Comment by Stephen-debug** [comment]
Thank you for attempting to help, good news I have figured it out.

Let me share my solution in case others find it useful. Admittedly it was a rooky mistake I only started making sites 2 weeks ago sorry to waste your time.

Basically I neglected to give the Loop template a condition for where to display it, I needed to set it as a Post. Once I did this I just had to go back and change my Jet Smart Filters checkbox filter settings to direct to Elementor Pro Posts.
